css/index.css

```diff
@@ -1 +1,310 @@
+  vertical-align: top;
+  font-size: 77px;
+  margin-top: 120px;
+  text-align: center;
+  padding: 0px 75px;
+}
+
+.cta .cta-text button {
+  background: white;
+  vertical-align: top;
+  margin-top: 6px;
+  padding: 8px 44px;
+  font-size: 17px;
+  border: 1px solid black;
+  cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+.cta img {
+  display: inline-block;
+  margin-top: 69px;
+}
+
+.main-content {
+  border-top: 1px solid black;
+  margin-top: 76px;
+  padding-top: 39px;
+}
+
+.main-content .top-content {
+  padding: 0 10px;
+}
+
+.main-content .top-content .text-content {
+  width: 410px;
+  display: inline-block;
+  margin-right: 34px;
+  padding-bottom: 20px;
+}
+
+.main-content .top-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.main-content .middle-img {
+  margin-top: 45px;
+}
+
+.main-content .bottom-content {
+  margin-top: 37px;
+  padding: 0 10px;
+}
+
+.main-content .bottom-content .text-content {
+  width: 275px;
+  display: inline-block;
+  margin-right: 12px;
+}
+
+.main-content .bottom-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.contact {
+  margin-top: 57px;
+  border-top: 1px solid black;
+  padding: 39px 10px 0;
+}
+
+.contact p {
+  margin: 20px 0;
+}
+
+footer {
+  margin: 32px 0;
+  text-align: center;
+  font-size: 14px;
+}
+
+@media (max-width: 800px) {
+    .container {
+        width: 100%;
+    }
+    header nav {
+      width: 768px;
+      padding: 0 30px;
+    }
+    header .logo {
+      order: -1;
+      display: block;
+      margin: 30px auto 0;
+    }
+    .cta {
+     justify-content: center;
+    }
+    .cta .cta-text {
+      margin: 0;
+    }
+    .cta img {
+      display: none;
+    }
+    .main-content .middle-img {
+       width: 100;
+
+    }
+    .main-content {
+      margin: 75px 20px 35px;
+   }
+
+   .text-content {
+       margin: 0 10px;
+   }
+
+   .contact {
+       margin: 0 30px;
+   }
+}
+
+
+
+@media (max-width: 500px) {
+  
+  }
+  header {
+      justify-content: center;
+  }
+
+  header nav {
+      width: 100%;
+      flex-direction: column;
+      margin: 0;
+      padding: 0;
+  }
+  header .logo {
+    display: flex;
+    margin: 0 auto;
+    align-items: flex-start;
+  }
+  header nav a {
+      width: 100%;
+      display: block;
+      text-align: center;
+      padding: 20px 0;
+      font-size: 20px;
+      border-bottom: 1px solid gray;
+  }
+  
+  header nav a:first-child {
+      border-top: 1px solid gray;
+  }
+
+  header nav a:hover {
+      background: gray;
+  }
+  
+  .cta {
+      margin-top: 50px;
+      flex-flow: column;
+      align-items: center;
+  }
+
+  .cta img {
+      max-width: 80%;
+  }
+
+  .main-content .top-content {
+      justify-content: space-around;
+      flex-direction: column;
+      height: 300px;
+      padding-bottom: 28px;
+  }
+
+  .main-content .bottom-content {
+      flex-wrap: wrap;
+      align-content: space-between;
+      height: 440px;
```

Comment:
this hard-coded height is causing a lot of whitespace on the page, is that intended? Try removing the height in DevTools, see how it looks!